run-test-in-docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
This script was not intended to be checked in.

It might make sense to check one in in the future, but should be done in a separate PR and should copy in local files instead of cloning from github.